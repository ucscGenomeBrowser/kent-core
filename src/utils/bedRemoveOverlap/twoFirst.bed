chr1 100 200 big
chr1 199 202 small
