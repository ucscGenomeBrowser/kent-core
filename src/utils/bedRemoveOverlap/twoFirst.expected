chr1 100 200 big
